pkg/provisioner/demo/demo.go

```diff
@@ -305,44 +278,18 @@ func (p *demoProvisioner) Detach() (result provisioner.Result, err error) {
 // PowerOn ensures the server is powered on independently of any image
 // provisioning operation.
 func (p *demoProvisioner) PowerOn(_ bool) (result provisioner.Result, err error) {
-	hostName := p.objectMeta.Name
-	switch hostName {
-	default:
-		return result, nil
-	}
-
-	// p.log.Info("ensuring host is powered on")
-
-	// if !p.host.Status.PoweredOn {
-	// 	p.host.Status.PoweredOn = true
-	// 	result.Dirty = true
-	// 	return result, nil
-	// }
-
-	// return result, nil
+	p.log.Info("powering on host")
+	return result, nil
```

Comment:
I was not sure how to get rid of this default case only issue, hence I am putting a log message, open for discussion.

Reply:
same as above